### ISSUE TITLE
Run LightHouse accessibility on all pages 

### ISSUE THREAD

**Issue by arslanashraf7** [mitodl/mitxonline#171]
This ticket is the last part of #150 epic.

**Note:** This should be done at the finalizing stage of all the accessibility fixes which means that we check the accessibility once the fixes for https://github.com/mitodl/mitxonline/issues/153, https://github.com/mitodl/mitxonline/issues/154, https://github.com/mitodl/mitxonline/issues/155 are done:

- [ ] Use the Lighthouse tool to review the accessibility of all other page templates and make any necessary changes

**Comment by pdpinch** [comment]
@arslanashraf7 what remains to do on this issue?

**Comment by arslanashraf7** [comment]
**_Update:_** At the time of creating this ticket there were multiple accessibility fixes making their way out and we could identify only two pages that needed accessibility fixes were the `Dashboard & Product Detail` Page which were fixed in https://github.com/mitodl/mitxonline/pull/176.

Now I re-ran the accessibility on `Auth, Registration, Dashboard, Home, Product Detail, Profile` and the accessibility is 💯. So, I think at this point we can close this issue and if we see other accessibility issues in the future they should be done under another ticket.
